x-pack/solutions/search/plugins/search_synonyms/server/config.ts

```diff
@@ -9,7 +9,7 @@ import { schema, TypeOf } from '@kbn/config-schema';
 import { PluginConfigDescriptor } from '@kbn/core/server';
 
 const configSchema = schema.object({
-  enabled: schema.boolean({ defaultValue: false }),
+  enabled: schema.boolean({ defaultValue: true }),
```

Comment:
If we do this here then I _think_ we need to explicitly disable the plugin in the other serverless solutions. Like we do for other [plugins](https://github.com/elastic/kibana/blob/main/config/serverless.oblt.yml#L9-L12)

Reply:
But we may not have to do that anymore soon with the distributions, but I'm not following that closely enough to know for sure. cc: @afharo

Reply:
tested locally, serverless.yml is working as expected (false by default)

Reply:
Ok, this was causing issues with FTRs. Updating them